### PR TITLE
perf: sync and update permissions.json file

### PR DESCRIPTION
**Pull request by SLdragon**
- Update permissions.json file to address these bugs: 
  - [Bug 15127844](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/15127844)
  - https://github.com/OfficeDev/TeamsFx/issues/4514
- Update error message to tell user to use uuid instead

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -673,7 +673,7 @@
   "error.aad.manifest.AADManifestIssues": "AAD manifest has following issues that may potentially break the Teams App:\n",
   "error.aad.manifest.UnknownResourceAppId": "Unknown resourceAppId %s",
   "error.aad.manifest.UnknownResourceAccessType": "Unknown resourceAccess: %s",
-  "error.aad.manifest.UnknownResourceAccessId": "Unknown resourceAccess id: %s",
+  "error.aad.manifest.UnknownResourceAccessId": "Unknown resourceAccess id: %s, if you're using human-readable string as resourceAccess id, please try to use uuid instead.",
```
Comment (adashen) [inline packages/fx-core/resource/package.nls.json:676]: do we have more suitable wording for "human readable string"?

File: packages/fx-core/src/plugins/resource/aad/permissions/permissions.json [hunk 2]
```diff
@@ -53,1464 +53,1453 @@
           "allowedMemberTypes": [
             "Application"
           ],
-          "description": "Allow the app to evaluate the inputs provided against the Data Loss Prevention policies and recommend applicable actions, without a signed-in user.",
-          "displayName": "Evaluate Data Loss Prevention policy",
-          "id": "ba7b8302-40ad-475c-a768-5b990aa1dba1",
+          "description": "Allows\u00a0the\u00a0app\u00a0to\u00a0read\u00a0subject\u00a0rights requests\u00a0without a\u00a0signed-in\u00a0user.",
```
Comment (adashen) [inline packages/fx-core/src/plugins/resource/aad/permissions/permissions.json:56]: is this change as desired?

Comment (SLdragon) [reply packages/fx-core/src/plugins/resource/aad/permissions/permissions.json:56]: Yes, it is come from graph api response, and I have created an issue here:
https://github.com/microsoftgraph/microsoft-graph-docs/issues/17943

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/resource/package.nls.json:676]: how about scope string?

**Review by KennethBWSong — APPROVED** [review]

**Pull request merged**